pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -239,30 +240,29 @@ func evictPodsFromTargetNodes(
 		totalMem += ((float64(memPercentage) * float64(nodeCapacity.Memory().Value())) / 100)
 	}
 
-	klog.V(1).Infof("Total capacity to be moved: CPU:%v, Mem:%v, Pods:%v", totalCPU, totalMem, totalPods)
-	klog.V(1).Infof("********Number of pods evicted from each node:***********")
+	klog.V(1).InfoS("Total capacity to be moved", "CPU", totalCPU, "Mem", totalMem, "Pods", totalPods)
+
+	klog.V(1).InfoS("********Number of pods evicted from each node:***********")
 
 	for _, node := range targetNodes {
 		nodeCapacity := node.node.Status.Capacity
 		if len(node.node.Status.Allocatable) > 0 {
 			nodeCapacity = node.node.Status.Allocatable
 		}
-		klog.V(3).Infof("evicting pods from node %#v with usage: %#v", node.node.Name, node.usage)
-
+		klog.V(3).InfoS("evicting pods from node", "node", klog.KObj(node.node))
```

Comment:
could you add the `node.usage` too? Looks like that got dropped (or was that intentional?)

Reply:
no it wasn't, thanks